docs/DEV.md

```diff
@@ -361,3 +361,26 @@ The default Basic Auth username is `admin`, which is written to `/var/lib/faasd/
 * `faasd install` - install faasd and containerd with systemd, this must be run from `$GOPATH/src/github.com/openfaas/faasd`
 * `journalctl -u faasd -f` - faasd service logs
 * `journalctl -u faasd-provider -f` - faasd-provider service logs
+
+#### Uninstall
+
+* Stop faasd and faasd-provider
```

Comment:
I think you missed `systemctl disable faasd` and `systemctl disable faasd-provider` and `systemctl disable containerd`

We should also probably remove the `containerd` binary.

Reply:
Done